modules/candidate_parameters/js/.gitignore

```diff
@@ -0,0 +1 @@
+index.js
```

Comment:
This should be '*', there's no files there that aren't auto generated (it's already * on my computer, I think it just wasn't added because it ignored itself.)